example/android/SNetDemo/semantic-segmentation-api/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
 }
 
 tasks.register('getImageSegmentationApi', io.singularitynet.sdk.gradle.GetSingularityNetServiceApi) {
+    ethereumJsonRpcEndpoint = new URL('https://mainnet.infura.io/v3/e7732e1f679e461b9bb4da5653ac3fc2')
```

Comment:
It is not required anymore because this value is read from `ethereum.properties`
```suggestion
```